content/docs/user-guide/assets/asset-types.md

```diff
@@ -0,0 +1,32 @@
+| `.bmp`</br>`.gif`</br>`.jpg`</br>`.jpeg`</br>`.png`</br>`.tga`</br>`.tif`</br>`.tiff` | Image file formats commonly used as textures for materials and UI elements. Image source assets generate a streaming image product asset and might generate several mipmap product assets if mipmap generation is enabled. | `.streamingimage`</br>`.imagemipchain` |
+| `.dds` | A specialized compressed image format most often used for normal maps and texture atlases. | `.streamingimage` |
+| `.exr` | High dynamic range (HDR) images used for Image Based Lighting (IBL) and skyboxes. HDR image source assets generate a streaming image product asset and might generate several mipmap product assets if mipmap generation is enabled. | `.streamingimage`</br>`.imagemipchain` |
+| `.fbx` | A scene file format that stores complete 3D scenes including meshes, skeletons, skin weights, animation, and materials. 3D scene source assets generate various product assets depending on the contents of the source asset and the data specified for processing. 3D scene files usually contain at least one mesh, and each mesh can produce over a dozen product assets including a model, a buffer for each of the model's vertex attributes, materials, and PhysX meshes. | `.azmodel`</br>`.azlod`</br>`.azbuffer`</br>`.actor`</br>`.skinmeta`</br>`.motion`</br>`.pxmesh`</br>`.azmaterial` |
+| `.gltf`</br>`.glb` |  An open source scene transport format that stores complete 3D scenes including meshes, skeletons, skin weights, animation, textures, and materials. 3D scene source assets generate various product assets depending on the contents of the source asset and the data specified for processing. 3D scene files usually contain at least one mesh, and each mesh can produce over a dozen product assets including a model, a buffer for each of the model's vertex attributes, materials, and PhysX meshes. | `.azmodel`</br>`.azlod`</br>`.azbuffer`</br>`.actor`</br>`.skinmeta`</br>`.motion`</br>`.pxmesh`</br>`.azmaterial` |
+| `.gradimagesettings` | Stores properties and values that define an image gradient. | `.gradimagesettings` |
+| `.inputbindings` | Binds input from keyboards, gamepads, touchscreens, and other devices to runtime events. | `.inputbindings` |
+| `.postFXLayerCategories` | A list of key value pairs that define layers for Post FX assignments. | `.postFXLayerCategories` |
+| `.prefab` | A spawnable asset container that can include entities and/or other prefabs. | `.spawnable` |
+| `.stl` | A simple mesh format used in computer aided design (CAD) and 3D printing. Unlike other 3D scene formats, `.stl` assets can only contain a simple model with limited vertex attributes. | `.azmodel`</br>`.azlod`</br>`.azbuffer`</br>`.pxmesh` |
+| `.material` | Stores the properties and values, including texture assignments, that define the look of a surface for rendering. | `.azmaterial` |

```

Comment:
The alphabetized order is interrupted starting with `.material` appearing after `.stl`. Can you help those of us who are bothered by this and fix the alphabetization? :P